### PR TITLE
New format: Japanese Apache Tomcat

### PR DESCRIPTION
**Pull request by Chaphasilor**
Wasn't posted on the sub, but I got ahold of the URL and thought it would be a nice exercise to add support for it!

URL is http://126.79.119.89:8181/, but it seems to be down a lot (every evening?)...  
I managed to save the HTML source earlier today though, so I could add tests, etc.

However, you should probably see if the OD comes back online again and test it with the live OD :)

PS: You might wanna merge #74 before this PR, because I was brain-afk and added the comments from it to the other branch 😅

### CODE REVIEW

File: OpenDirectoryDownloader/DirectoryParser.cs [hunk 1]
```diff
@@ -2066,9 +2066,9 @@ private static HeaderInfo GetHeaderInfo(IElement header)
 
             headerName = headerName.ToLower();
 
-            headerName = Regex.Replace(headerName, @"[^a-zA-Z0-9\s一-龥äöüÄÖÜß]", string.Empty);
+            headerName = Regex.Replace(headerName, @"[^\u00BF-\u1FFF\u2C00-\uD7FF\w]", string.Empty);
```
Comment (KoalaBear84) [inline OpenDirectoryDownloader/DirectoryParser.cs:2069]: Do you have more info about this other regex?

Comment (Chaphasilor) [reply OpenDirectoryDownloader/DirectoryParser.cs:2069]: The problem with the old RegEx essentially was that the ranges you specified (A-z,a-z, etc.) didn't include some japanese characters (chinese Hanzi seemed to work though). The new RegEx uses unicode ranges instead, in order to include most every forgein characters.

With the old pattern, some parts of the column headings were ignored and the type detection didn't work.

I can link the thread where I got it from once I get to a computer :)

Comment (KoalaBear84) [reply OpenDirectoryDownloader/DirectoryParser.cs:2069]: OK, np, I'll see it later.

Comment (Chaphasilor) [reply OpenDirectoryDownloader/DirectoryParser.cs:2069]: see the second answer here: https://stackoverflow.com/questions/3009993/what-would-be-regex-for-matching-foreign-characters

Comment (KoalaBear84) [reply OpenDirectoryDownloader/DirectoryParser.cs:2069]: OK, thanks, good 👍

**Comment by KoalaBear84** [comment]
https://github.com/KoalaBear84/OpenDirectoryDownloader/releases/tag/v1.9.4.4

**Pull request merged**